copy/copy.go

```diff
@@ -155,7 +160,7 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		}
 	}
 
-	canModifyManifest := len(sigs) == 0
+	canModifyManifest := len(sigs) == 0 || options.RemoveSignatures
```

Comment:
Why is this necessary? Per
```go
var sigs [][]byte
if options != nil && options.RemoveSignatures {
    sigs = [][]byte{}
```
above, this seems redundant.

(Also, `options` can be `nil`.)